openam-server-only/src/main/resources/services/idRepoService.xml

```diff
@@ -38,8 +39,8 @@
         <Schema
             i18nFileName="amIdRepoService"
             revisionNumber="30"
-            i18nKey="">
-
+            i18nKey=""
+            resourceName="datastores">
```

Comment:
I am not sure if I want to introduce inconsistency between the technical service name and its resource name. I would prefer to have it is `id-repos`. I am OK with "datastores" on the UI level though as there are other examples where we have such inconsistency (e.g. realm vs. organization or application vs. policy sets).

Reply:
I have renamed the endpoint to `id-repos`.